expression/builtin_other.go

```diff
@@ -215,3 +217,30 @@ func (b *builtinBitCountSig) eval(row []types.Datum) (d types.Datum, err error)
 	d.SetInt64(count)
 	return d, nil
 }
+
+// for plan cache of prepared statements
+type getParamFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *getParamFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
```

Comment:
do the `c.verifyArgs(args)` operation first

Reply:
Thank you for the sugguestion. I've changed the codes as follows:

`+//TODO: more typed functions will be added when typed parameters are supported.`
` func (c *getParamFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {`
`-  bt := &builtinGetParamSig{newBaseBuiltinFunc(args, ctx)}`
`-  bt.deterministic = false`
`-  bt.deferred = true`
`-  return bt.setSelf(bt), errors.Trace(c.verifyArgs(args))`
`+  if err := c.verifyArgs(args); err != nil {`
`+      return nil, errors.Trace(err)`
`+  }`
`+  bf := newBaseBuiltinFuncWithTp(args, ctx, tpString, tpInt)`
`+  sig := &builtinGetParamStringSig{baseStringBuiltinFunc{bf}}`
`+  sig.deterministic = false`
`+  sig.deferred = true`
`+  return sig.setSelf(sig), nil`
` }`